components/org.wso2.carbon.identity.application.authenticator.hypr/src/main/java/org/wso2/carbon/identity/application/authenticator/hypr/HyprAuthenticator.java

```diff
@@ -169,7 +170,13 @@ protected void initiateAuthenticationRequest(HttpServletRequest request, HttpSer
                                                  AuthenticationContext context) throws AuthenticationFailedException {
 
         try {
-            redirectHYPRLoginPage(response, context, null);
+            if (context.getLastAuthenticatedUser() != null) {
```

Comment:
should we check this in `process` method?

Reply:
it is better to check in the process method. moved with 0bfcecd